monitoring/defender/util.ts

```diff
@@ -13,3 +15,22 @@ export const defenderNetworkToSupportedNetwork = (networks: Network) => {
       throw new Error(`Unsupported network ${networks}`);
   }
 };
+
+export const readFileAndReplace = (
+  filePath: string,
+  replaceMap: { [toReplace: string]: string }
+) => {
+  const buffer = fs.readFileSync(filePath);
+  let fileContent = buffer.toString();
+  Object.keys(replaceMap).forEach((key) => {
+    fileContent = fileContent.replace(key, replaceMap[key]);
+  });
+  return fileContent;
+};
```

Comment:
Have you checked that this works when it's executed in OZ Defender? I did not get it to function correctly this way. That is why I converted it into what is done here: https://github.com/gnosis/zodiac-safe-app/tree/master/packages/backend/lib/defender

Reply:
Yes, I tested also; you can see here that it is setting up  the variable properly  <img width="707" alt="image" src="https://user-images.githubusercontent.com/19823989/205292360-eee2c840-1f6e-484d-9f7c-7652aae4892d.png">